Assets/Resources/MagicItemTemplates.txt

```diff
@@ -1,1359 +1,3070 @@
 [
     {
-        "position": 4,
```

Comment:
I'm curious about the change from "position" to "index" here? This value is the byte position the record is found in the source binary file (from binaryReader.BaseStream.Position at export time) - not the sequential index of the record. I've used this "position" convention elsewhere as well, as it matches the StreamReader naming.

It should be functionally fine as I'm not consuming that value elsewhere, but I feel it's a bit less clear now. I was mainly just curious why the need to change?

Reply:
I created the MagicItemTemplates.txt file by using the "Generate Magic Item JSON" button in the editor, so that change was done automatically because it's "index" in the file. I looked at GitHub's blame page for MagicItemsFile.cs (https://github.com/Interkarma/daggerfall-unity/blame/8bd34943b3afb0248c5bc593f0808c4b4672e4a2/Assets/Scripts/API/MagicItemsFile.cs) and it looks like it's been that name for a long time now. Anyway, I don't have a preference about the name if you want to change it!

Reply:
That's interesting haha! I have no memory of changing that. :) I thought I exported that data from my offline test code, which is full of one-off code that never made it to git.